projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/r2/index.js

```diff
@@ -19,9 +19,9 @@ const PluginError = gutil.PluginError;
 const PLUGIN_NAME = 'gulp-R2-css';
 
 function gulpR2() {
-	const task = through.obj(function (file, enc, cb) {
+	const task = through.object(function (file, enc, callback) {
```

Comment:
This is `obj` to `object` is likely going to cause issues. `through.obj` is a method and so we can't change this.

Are we able to add a test case for this so that we don't emit errors for abbreviations like this?

Reply:
You might be able to check if its a property by doing something like

```js
    Identifier(node) {
        const isProperty = node === node.parent.property;
```